drivers/stm32_eth/stm32_eth.c

```diff
@@ -31,6 +31,16 @@ static mutex_t _tx = MUTEX_INIT;
 static mutex_t _rx = MUTEX_INIT;
 netdev_t *_netdev;
 
+#if IS_USED(MODULE_STM32_ETH_LINK_UP)
+#include "xtimer.h"
+
+#define TIMEOUT_US 1 * US_PER_SEC
```

Comment:
```suggestion
#define STM32_ETH_LINK_UP_TIMEOUT_US    (1UL * US_PER_SEC)
```

1. Though `stm32` is 32-bit, let's be consistent about literal typing
2. `TIMEOUT_MS` is a very generic name, that might also be used in one of the other excludes.
3. The parenthesis ensure that this value is used as a value in a term and not as a term that might need to be put in parenthesis.

Reply:
This is still unaddressed.

Reply:
Parts of this change request still missing or unaddressed. See https://github.com/RIOT-OS/RIOT/pull/14688#discussion_r469985259